src/cms/templates/pages/page_tree.html

```diff
@@ -86,11 +86,12 @@ <h1 class="heading">{% trans 'Pages' %}</h1>
             {% get_last_root_page pages as last_root_page %}
             {% recursetree pages %}
                 {% get_translation node language.slug as page_translation %}
+                {% get_depth_in node pages as page_depth %}
                 {% include "pages/page_tree_node.html" with page=node %}
                 {% if not node.is_leaf_node %}
                     {{children}}
                 {% endif %}
-                {% if node == last_root_page %}
+                {% if node == last_root_pages %}
```

Comment:
```suggestion
                {% if node == last_root_page %}
```
This was just a typo, right?